SETUP/db_schema.sql

```diff
@@ -11,7 +11,7 @@ CREATE TABLE `access_log` (
   `subject_username` varchar(25) NOT NULL default '',
   `modifier_username` varchar(25) NOT NULL default '',
   `action` varchar(16) NOT NULL default '',
-  `activity` varchar(10) NOT NULL default '',
+  `activity` varchar(32) NOT NULL default '',
```

Comment:
The commit message should say why the column needs to be widened.

Reply:
<nod> I'll do so after at the end of this CR since that will rebase all of the commits.